resources/installation.rb

```diff
@@ -38,6 +38,7 @@ def binary_basename
   remote_file archive_path do
     source format(attrs['archive_url'], version: new_resource.version, basename: basename)
     checksum attrs['archive_checksum'] || nil
+    cookbook attrs['archive_prefix'] || nil
```

Comment:
You sure about that? I don't see any `cookbook` attribute in https://docs.chef.io/resources/remote_file/

Reply:
We use a custom one.
PR abandonned anyway